addons/test_discuss_full/tests/test_performance.py

```diff
@@ -99,7 +99,7 @@ def test_init_messaging(self):
         self.maxDiff = None
         self.users[0].flush()
         self.users[0].invalidate_cache()
-        with self.assertQueryCount(emp=96):  # ent: 95
+        with self.assertQueryCount(emp=97):  # ent: 95
```

Comment:
The comment is probably obsolete?
But should probably be reverted (see previous comments).